src/test/java/TestSuite.java

```diff
@@ -0,0 +1,199 @@
+import com.google.gson.JsonSyntaxException;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.junit.platform.suite.api.IncludeTags;
+import org.junit.platform.suite.api.SelectClasses;
+import org.junit.platform.suite.api.Suite;
+import parser.JsonParser;
+import parser.NoSuchFileException;
+import parser.Parser;
+import shop.Cart;
+import shop.RealItem;
+import shop.VirtualItem;
+
+import java.io.File;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+@Suite
+@SelectClasses({JsonParserTest.class, RealItemTest.class, VirtualItemTest.class, CartTest.class})
+@IncludeTags({"JsonParserTest", "RealItemTest", "VirtualItemTest", "CartTest"})
```

Comment:
Keeping this approach will require to keep @SelectClasses. Otherwise, no tests will be found. But you're right, @IncludeTags is not necessary here so no need to keep both.